src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -0,0 +1,27 @@
+const MORPH_SUFFIX = 'morph_env=test';
+const DOMAIN_PREFIX = {
+  local: 'http://localhost.bbc.com:8080/ws/av-embeds',
+  test: 'https://www.test.bbc.co.uk/ws/av-embeds',
+  live: 'https://www.bbc.com/ws/av-embeds',
+};
+
+const embedUrl = (env, assetId, vpid, amp = false) => {
```

Comment:
This functions a bit hard to follow, can it be refactored at all to make it easier to follow?